qgreenland/util/config.py

```diff
@@ -15,9 +16,11 @@
 import fiona  # noqa: I100
 # HACK HACK HACK HACK HACK HACK HACK HACK HACK THIS IS A DUMB HACK HACK HACK
 import yamale
+from humanize import naturalsize
```

Comment:
How do we tell `mypy` to ignore 3rd party libraries that do not have type definitions?

Reply:
```
cd /home/trst2284/code/qgreenland && mypy --config-file=/home/trst2284/code/qgreenland/.mypy.ini /home/trst2284/code/qgreenland/qgreenland
qgreenland/util/config.py:19: error: Skipping analyzing 'humanize': found module but no type hints or library stubs
qgreenland/util/config.py:19: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 51 source files)
```

Sounds like I need to check out https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports :laughing: